prismic-model/src/parts/visual-story-body.ts

```diff
@@ -59,6 +59,20 @@ export default {
           isZoomable: boolean('Allow image to be zoomed to fill viewport?'),
         },
       }),
+      textAndIcons: slice('Text and icons', {
+        description: 'Side-by-side',
```

Comment:
Do we want to add something about limited image width and/or multiples to differentiate it from Text and image?